package.json

```diff
@@ -34,6 +34,8 @@
     }
   },
   "dependencies": {
+    "@analytics/google-analytics": "^0.5.2",
```

Comment:
Not sure why we'd want to pull in GA. Doing so will encourage people to avoid using a CDP (Customer Data Processor) like Segment, which is what most businesses will want anyway. On top of that Google Analytics is built more for Webpages and not Apps like this IMO. If users really want this they could add it on their own. Otherwise we can pull it and (a) still use GA because (b) users can customize if they like by adding this dependency.

Reply:
I don't really agree. Neither to I think it should be much of a concern for us wether we should encourage to use segment or not, nor do I think that Google Analytics is not suited for this kind of ecommerce platform, I see no reason to differentiate between app or Webpages here.

I'm fine with keeping it, as long as it doesn't load per default if no `GOOGLE_ANALYTICS_ID ` was set, which does not seem to be the case.